### DISCUSSION TITLE
Use a Deno module in a Node project

### DISCUSSION THREAD

**Discussion by hazae41** [denoland/deno#10505] [Q&A] [answered]
Hey, 

Is there a way to import a Deno module in a Node project?

Similar to https://deno.land/std/node but the other way around

For example, import something that uses `Deno.readTextFile` and/or some Deno std modules in Node

Does something like this exist? Thanks

**Comment by ry** [comment, marked as answer]
This is currently under active development: https://github.com/trgwii/deno.ns

**Reply by hazae41** [reply]
Thanks!

**Reply by elycheikhsmail** [reply]
I think is'nt good idea to try to run deno module from node js but you can use deno js directly to generate some code or manipulate some node project

**Reply by elycheikhsmail** [reply]
when deno std library is stable , I think we don't need to use nodejs for backend. but for frontend we will still need nodejs package for some periode.
